sql/mob_skill_lists.sql

```diff
@@ -2692,3 +2692,770 @@ INSERT INTO `mob_skill_lists` VALUES ('Alexander_LowHP',785,2145); -- Gospel of
+INSERT INTO `mob_skill_lists` VALUES ('Akash',808,909); -- glittering ruby
+INSERT INTO `mob_skill_lists` VALUES ('Akash',808,910); -- meteorite
+
+-- Adamastor
+INSERT INTO `mob_skill_lists` VALUES ('Adamastor',809,664); -- impact roar
+INSERT INTO `mob_skill_lists` VALUES ('Adamastor',809,665); -- grand slam
+INSERT INTO `mob_skill_lists` VALUES ('Adamastor',809,666); -- power attack
+
+-- Baba_Yaga
+INSERT INTO `mob_skill_lists` VALUES ('Baba_Yaga',810,2193); -- zephyr arrow
+INSERT INTO `mob_skill_lists` VALUES ('Baba_Yaga',810,2194); -- lethe arrows
+
+-- Briareus
+INSERT INTO `mob_skill_lists` VALUES ('Briareus',811,666); -- power attack
+INSERT INTO `mob_skill_lists` VALUES ('Briareus',811,1636); -- trebuchet
+-- INSERT INTO `mob_skill_lists` VALUES ('Briareus',811,2253); -- Meikyo Shisui
+INSERT INTO `mob_skill_lists` VALUES ('Briareus',811,2576); -- mercurial strike
+INSERT INTO `mob_skill_lists` VALUES ('Briareus',811,2578); -- colossal slam
+
+-- Carabosse
+INSERT INTO `mob_skill_lists` VALUES ('Carabosse',812,2193); -- zephyr arrow
+INSERT INTO `mob_skill_lists` VALUES ('Carabosse',812,2194); -- lethe arrows
+
+INSERT INTO `mob_skill_lists` VALUES ('Carabosse_Post_Benediction',813,2193); -- zephyr arrow  -- group 813 used only after benediction
+INSERT INTO `mob_skill_lists` VALUES ('Carabosse_Post_Benediction',813,2194); -- lethe arrows
+INSERT INTO `mob_skill_lists` VALUES ('Carabosse_Post_Benediction',813,2195); -- spring breeze 
+INSERT INTO `mob_skill_lists` VALUES ('Carabosse_Post_Benediction',813,2196); -- summer breeze
+INSERT INTO `mob_skill_lists` VALUES ('Carabosse_Post_Benediction',813,2197); -- autumn breeze
+INSERT INTO `mob_skill_lists` VALUES ('Carabosse_Post_Benediction',813,2198); -- winter breeze
+
+-- Chasmic Hornet
+INSERT INTO `mob_skill_lists` VALUES ('Chasmic_Hornet',814,335); -- pollen
+-- INSERT INTO `mob_skill_lists` VALUES ('Chasmic_Hornet',814,1847); -- frenzy pollen
+-- INSERT INTO `mob_skill_lists` VALUES ('Chasmic_Hornet',814,2608); -- terminal sting
+
+-- Dozing_Dorian
+INSERT INTO `mob_skill_lists` VALUES ('Dozing_Dorian',815,262); -- sheep charge
+INSERT INTO `mob_skill_lists` VALUES ('Dozing_Dorian',815,265); -- rage
+INSERT INTO `mob_skill_lists` VALUES ('Dozing_Dorian',815,1837); -- feeble bleat
+
+-- Grandgousier
+INSERT INTO `mob_skill_lists` VALUES ('Grandgousier',816,658); -- catapult
+INSERT INTO `mob_skill_lists` VALUES ('Grandgousier',816,662); -- lightning roar
+INSERT INTO `mob_skill_lists` VALUES ('Grandgousier',816,1636); -- trebuchet
+
+-- Hadhayosh
+INSERT INTO `mob_skill_lists` VALUES ('Hadhayosh',817,628); -- wild horn
+INSERT INTO `mob_skill_lists` VALUES ('Hadhayosh',817,629); -- thunderbolt
+INSERT INTO `mob_skill_lists` VALUES ('Hadhayosh',817,632); -- flame armor
+INSERT INTO `mob_skill_lists` VALUES ('Hadhayosh',817,633); -- howl
+INSERT INTO `mob_skill_lists` VALUES ('Hadhayosh',817,2390); -- accursed armor
```

Comment:
I think as long as there is no script this skill just get skipped in the mobs selection of tp skill anyway, so it should be fine like this. If I'm wrong we'd just need to comment it out till I fix the spike thing.